history-of-quakers/modernized/09-chapter-7.adoc

```diff
@@ -391,7 +410,9 @@ in the latter end of the year 1660, about the 44th year of his age.
 About two hours before his death, he spoke,
 in the presence of several witnesses these words:
 
-'`There is a spirit which I feel, that delights to do no evil, nor to revenge any wrong,
+[quote]
```

Comment:
Sounds good. I think I have the mindset of thinking the embedded things are _written_ things only, for some reason. Probably because most of them are? Idk. :)

Reply:
Your mindset is pretty much correct, this is sort of an outlier, so stick with your intuition going forward.  I think because someone wrote this down when he said it, and it was widely published and became one of the most treasured pieces of quaker literature, it felt like it deserved to be an embedded testimony. And I think we made it that in the Nayler book we have on the site too, so I sort of thought consistency might be good there. 

So, anyway, your mindset is correct. 👍